Userland/Libraries/LibVT/TerminalWidget.cpp

```diff
@@ -1135,6 +1140,54 @@ void TerminalWidget::update_paste_action()
     m_paste_action->set_enabled(GUI::Clipboard::the().mime_type().starts_with("text/") && !GUI::Clipboard::the().data().is_empty());
 }
 
+void TerminalWidget::set_color_scheme(const StringView& name)
+{
+    if (name.contains('\\')) {
```

Comment:
Pretty sure you can have a backslash in Unix filenames, this ain't Windows :)
Probably meant to use a forward slash here?